docs/zkapps/advanced-snarkyjs/permissions.mdx

```diff
@@ -12,62 +12,65 @@ zkApps can now be deployed to Berkeley Testnet.
 
 # Permissions
 
-Permissions are an integral part of zkApp development - they determine who has the authority to interact and make changes to a specific part of a smart contract.
-Naturally, it's important for every smart contract to have a set of sound permissions to prevent any attacks or security holes.
-Permissions live on-chain, that means they are a part of the account representation on the network, and are checked every time an account updates tries to interact with an account.
+Permissions are an integral part of zkApp development because they determine who has the authority to interact and make changes to a specific part of a smart contract.
+Naturally, every smart contract must have proper permissions to prevent attacks or security holes.
+Permissions live on-chain, which means they are a part of the account representation on the network. Permissions are checked every time an account update tries to interact with an account.
 
 ## Types of Permissions
 
+There are 11 different types of permissions that a developer can access and adjust to guard a zkApp account:
 
-Currently, there are 11 different types of permissions that guard a zkApp account and a developer can access and adjust.
+- `editState`: The permission corresponding to the eight state fields of on-chain state that are associated with an account. This permission describes how the on-chate state fields can be manipulated.
```

Comment:
`The permission describing how the zkApp account's eight on-chain state fields are allowed to be manipulated.`